bin/hdfcoinc/pycbc_multiifo_sngls_findtrigs

```diff
@@ -31,59 +35,126 @@ parser.add_argument("--statistic-keywords", nargs='*',
                          "the statistic class when it is initialized. Should "
                          "be given in format --statistic-keywords "
                          "KWARG1:VALUE1 KWARG2:VALUE2 KWARG3:VALUE3 ...")
-parser.add_argument("--template-fraction-range", default="0/1",
-                    help="Optional, analyze only part of template bank. Format"
-                         " PART/NUM_PARTS")
```

Comment:
Wait. We definitely need these 3 options no? (Well maybe we don't need the randomize one, but it does help balance load)